backend/api/swagger/run.swagger.json

```diff
@@ -461,9 +461,6 @@
             "$ref": "#/definitions/apiRunMetric"
           },
           "description": "Output. The metrics of the run. The metrics are reported by ReportMetrics\nAPI."
```

Comment:
The unit tests for the experiment client wrapper will fail with this change. They need to be updated.